deny.toml

```diff
@@ -9,12 +10,11 @@ targets = [
 ]
 
 [licenses]
-unlicensed = "deny"
+version = 2
 allow = [
     "Apache-2.0",
     "MIT",
 ]
-copyleft = "deny"
```

Comment:
For version 2, `unlicensed = "deny"` and `copyleft = "deny"` are the defaults; see https://github.com/EmbarkStudios/cargo-deny/pull/611